src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Company.WebApplication1.http

```diff
@@ -0,0 +1,6 @@
+@Company.WebApplication1_HostAddress = http://localhost:5000
```

Comment:
I don't think this variable name would be replaced so it might be better to use something more generic since the project is almost certainly not going to be called `Company.WebApplication1` when someone uses the template ;)

... says he who is now up to `WebApplication1065` in his source directory. ;)